modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/Menu.tsx

```diff
@@ -118,7 +118,7 @@ export const DropdownMenu = ({
       case 'Enter':
         event.preventDefault();
         if (item.onAction) {
-          item.onAction(item);
+          item.onAction(item.link);
```

Comment:
At first I thought this was a breaking change, but after further inspection, it wouldn't have worked at all before (item was a string, and Strings have a deprecated `link` function`, which is what it was returning), so it's just a bug fix.